### ISSUE TITLE
Tag a 2.0 release

### ISSUE THREAD

**Issue by Redoubts** [NvChad/NvChad#1821]
I see the branch, but do you plan to tag a commit as 2.0? I think that would push notifications of the new release via GitHub.

**Comment by siduck** [comment]
hi @Redoubts I dont have much idea bout tags, would it create new release? Wouldnt I have to keep updating tags if there were bug fixes?

**Comment by Redoubts** [comment]
Yeah, I guess it depends how you feel about it. I only bring it up since I saw there was InitialChad and MegaChad here: https://github.com/NvChad/NvChad/releases; and was excited to see GigaChad. ([did you change that page?](https://webcache.googleusercontent.com/search?q=cache:VvBf0_n-7ooJ:https://github.com/NvChad/NvChad/releases))

idk what expectations you wanna set for further releases. I suppose you could just tag 2.0.0 and claim to be rolling release after. My only interest in this is I set github to watch releases on this project, and I suspect a few others have as well. I'm sure it would only help with getting the word out

**Comment by Redoubts** [comment]
Or maybe just the 130 prying eyes.
